### PR TITLE
Create doi

### PR DESCRIPTION
**Pull request by sri2k1us**
- On top of #283 
- Allow Admin to create DOI.

### CODE REVIEW

File: public/static/locales/en/doi.json [hunk 1]
```diff
@@ -2,6 +2,9 @@
     "cancel": "Cancel",
     "comments": "Comments",
     "createDoi": "Create DOI",
+    "createDoiPrompt": "Do you want to create DOI now?",
```
Comment (psarando) [inline public/static/locales/en/doi.json:5]:
```suggestion
    "createDoiPrompt": "Do you want to create a DOI now?",
```

**Review by psarando — APPROVED** [review]
LGTM 👍

**Comment by sri2k1us** [comment]
Thanks for the review. Additional reviews welcome.

**Comment by ash3rz** [comment]
(late review, but) LGTM :shipit:

**Pull request merged**